docs/source/installation.md

```diff
@@ -10,6 +10,33 @@ Strongly recommended; although this project is still in beta state.
 pip install moldrug
 ```
 
+````{admonition} Linux 🐧
+:class: Tip
+
+Recent Linux distributions require creation of virtual environment before calling `pip`.  
+Additionally the default Python might be too new for Moldrug (Python 3.12 is not yet supported).
```

Comment:
(...) Additionally the default Python might be too new for Moldrug. Python 3.12 is not yet supported (see pull requests [#9](https://github.com/ale94mleon/moldrug/pull/9) and [#10](https://github.com/ale94mleon/moldrug/pull/10)).